### ISSUE TITLE
CR fails on empty source files

### ISSUE THREAD

**Issue by abingham** [sixty-north/cosmic-ray#199]
It appears that we might have a problem parsing empty source files. To see this, try mutating a module for which the source file is empty. You should see something like:
```
Unable to read source for module <module 'intrapackage_deps.a' from '/Users/sixtynorth/projects/sixty-north/cosmic-ray/test_project/intrapackage_deps/a.py'>
Traceback (most recent call last):
  File "/Users/sixtynorth/.virtualenvs/cosmic-ray/bin/cosmic-ray", line 9, in <module>
    load_entry_point('cosmic-ray==1.0.0a1', 'console_scripts', 'cosmic-ray')()
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/cli.py", line 380, in main
    sys.exit(handler(sub_config))
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/cli.py", line 169, in handle_init
    timeout)
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/commands/init.py", line 21, in init
    counts = cosmic_ray.counting.count_mutants(modules, operators)
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/counting.py", line 54, in count_mutants
    for m in modules)
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/counting.py", line 47, in <dictcomp>
    mod: dict(
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/counting.py", line 54, in <genexpr>
    for m in modules)
  File "/Users/sixtynorth/projects/sixty-north/cosmic-ray/cosmic_ray/parsing.py", line 30, in get_ast
    source = inspect.getsource(module)
  File "/usr/local/Cellar/python3/3.5.0/Frameworks/Python.framework/Versions/3.5/lib/python3.5/inspect.py", line 943, in getsource
    lines, lnum = getsourcelines(object)
  File "/usr/local/Cellar/python3/3.5.0/Frameworks/Python.framework/Versions/3.5/lib/python3.5/inspect.py", line 930, in getsourcelines
    lines, lnum = findsource(object)
  File "/usr/local/Cellar/python3/3.5.0/Frameworks/Python.framework/Versions/3.5/lib/python3.5/inspect.py", line 761, in findsource
    raise OSError('could not get source code')
OSError: could not get source code
```

**Comment by atodorov** [comment]
FYI, we've dealt with this before. It is a known issue with Python:
https://github.com/sixty-north/cosmic-ray/blob/master/cosmic_ray/parsing.py#L32

**Comment by abingham** [comment]
Ah right, I thought this bug sounded familiar. Perhaps we have to manually check if the file is empty when we see an OSError.

**Comment by abingham** [comment]
Fixed on 60256dcc643aca30d4f475e3cf77a6dc01bdb587
